examples/build/ex_ppro.py

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Again, the explicit name `ex_papilio_pro` is preferred, it is not something that is typed a bunch.